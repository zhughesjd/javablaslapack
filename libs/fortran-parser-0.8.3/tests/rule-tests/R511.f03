
! Testing explicit_shape_spec_list.
common foo(1,2,3), bar(4:5,6)
end
