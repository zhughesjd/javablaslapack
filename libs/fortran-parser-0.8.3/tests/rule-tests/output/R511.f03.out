1101:begin
558:list-begin
511:list-begin
406 1 null
511 false
406 2 null
511 false
406 3 null
511 false
511:list 3
511:list-begin
406 4 null
406 5 null
511 true
406 6 null
511 false
511:list 2
558:list 2
1103
1101 false false false
